Resources/Prototypes/Entities/Mobs/Player/mothershipcore.yml

```
@@ -0,0 +1,195 @@
+- type: startingGear
+  id: MothershipCoreGear
+  inhand:
+  - DoorRemoteXenoborg
+  - Omnitool
```

Any chance this could be changed to
```suggestion
  - JawsOfLife
  - PowerDrill
```

---

the core only has 4 hands...

but I suppose I can leave them with a toolbox in the mothership with the test of the tools

---

oh right, I had it in my mind that it was like a borg with modules

---

there is a PR that plans to change how modules work by giving borgs hands with whitelist, I could change the mothership to use modules and have some hands free to pick up brains and materials and modules